src/main/java/seedu/mark/ui/MainWindow.java

```diff
@@ -306,4 +328,97 @@ private CommandResult executeCommand(String commandText) throws CommandException
             throw e;
         }
     }
+
+    /**
+     * Compares two time in hours.
+     *
+     * @param before the time that is before.
+     * @param after the time that is after.
+     * @return the difference of two time in hour.
+     */
+    private long compareHour(LocalDateTime before, LocalDateTime after) {
+        return Duration.between(before, after).toHours();
+    }
+
+    /**
+     * Compares two time in minutes.
+     *
+     * @param before the time that is before.
+     * @param after the time that is after.
+     * @return the difference of two time in minute.
+     */
+    private long compareMinute(LocalDateTime before, LocalDateTime after) {
+        return Duration.between(before, after).toMinutes();
+    }
+
+    /**
+     * Creates a notification for a specific reminder.
+     *
+     * @param reminder the reminder that is used to create notification.
+     * @return the notification for reminder.
+     */
+    private Notifications getNoti(Reminder reminder) {
+        Notifications noti = Notifications.create();
+        noti.title("Reminder Notification");
+
+
+        String remindMessage = reminder.getNote().toString() + " " + reminder.getUrl().value + "\n"
+                + "Due at : " + reminder.getFormattedTime();
+        noti.text(remindMessage);
+        Image image = new Image("/images/bell.png");
+        ImageView imageView = new ImageView(image);
+        imageView.setFitHeight(50);
+        imageView.setFitWidth(50);
+        noti.graphic(imageView);
+        noti.hideAfter(javafx.util.Duration.seconds(60));
+        noti.position(Pos.BOTTOM_RIGHT);
+
+        return noti;
+    }
+
+    /**
+     * Displays the reminder notification.
+     */
+    private void displayReminderMessage() {
+
+        TimerTask task = new TimerTask() {
+            @Override
+            public void run() {
+                LocalDateTime now = LocalDateTime.now();
+
+                try {
+                    for (int i = 0; i < reminders.size(); i++) {
+                        Reminder reminder = reminders.get(i);
+                        LocalDateTime remindTime = reminder.getRemindTime();
+                        System.out.println(compareHour(now, remindTime));
+                        System.out.println(now.isBefore(remindTime) && compareHour(now, remindTime) < 5);
```

Comment:
Can you remove all the `System.out.println` statements? If needed, they can be replaced by logging using `logger.fine` or `logger.info` (depending on how important it is).

Reply:
It is just testing methods, I'll move it before merge.